packages/api-client/src/components/DataTable/DataTableCell.vue

```diff
@@ -11,7 +12,12 @@ withDefaults(
 <template>
   <component
     :is="is"
-    class="min-h-8 min-w-8 border-l-0 border-t-0 border-b-1/2 border-r-1/2 flex text-sm last:border-r-0 group-last:border-b-transparent p-0 m-0 relative"
+    :class="[
+      tw(
+        'min-h-8 min-w-8 border-l-0 border-t-0 border-b-1/2 border-r-1/2 flex text-sm last:border-r-0 group-last:border-b-transparent p-0 m-0 relative',
+        $attrs.class as string,
```

Comment:
use tw-merge to override classes

Reply:
Do we actually want to `as string`? tw takes a few different options (also Vue will anything and smush it onto the class so we might need `inheritAttrs: false`)?

![Cursor-2024-11-26-14-23-45@2x](https://github.com/user-attachments/assets/8594b29c-697c-402b-b41c-ffd80e285e6e)

Reply:
aha no as string was just go get it to work. It comes in as any so it doesn't make a difference what we put. We can do it for posterity's sake